public/app/features/manage-dashboards/components/PublicDashboardListTable.tsx

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/css';
+import React, { FC, useState } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { Link, ButtonGroup, LinkButton, Icon, Tag, useStyles2 } from '@grafana/ui';
+import { getConfig } from 'app/core/config';
+
+export interface ListPublicDashboardResponse {
+  uid: string;
+  accessToken: string;
+  dashboardUid: string;
+  title: string;
+  isEnabled: boolean;
+}
+
+export const listPublicDashboardsUrl = () => {
+  return `/api/dashboards/public`;
+};
+export const getPublicDashboards = async () => {
+  const resp: ListPublicDashboardResponse[] = await getBackendSrv().get(listPublicDashboardsUrl());
+  return resp.sort((a, b) => Number(b.isEnabled) - Number(a.isEnabled));
+};
+
+export const viewPublicDashboardUrl = (accessToken: string): string => {
+  return `${getConfig().appUrl}public-dashboards/${accessToken}`;
+};
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    link: css`
+      color: ${theme.colors.primary.text};
+      text-decoration: underline;
+      margin-right: ${theme.spacing()};
+    `,
+  };
+}
+
+export const ListPublicDashboardTable: FC = () => {
+  const styles = useStyles2(getStyles);
+  const [publicDashboards, setPublicDashboards] = useState<ListPublicDashboardResponse[]>([]);
+
+  useAsync(async () => {
+    const publicDashboards = await getPublicDashboards();
+    setPublicDashboards(publicDashboards);
+  }, [setPublicDashboards]);
```

Comment:
useAsync func gives you loading and error flags (such as we talked with RTK Query) https://github.com/streamich/react-use/blob/master/docs/useAsync.md.
would be good to add a spinner for loading, for example. 
If we make a refactor for adding RTK Query, this loading state will be already implemented, so less work

Reply:
Adding to refactor : )

Reply:
https://github.com/grafana/grafana-partnerships-team/issues/408